Service/NavitiaService.php

```diff
@@ -186,9 +186,12 @@ private function getApiResponse(
             try {
                 return $this->cache->getCachedItem($cacheKey);
             } catch (CacheItemNotFoundException $e) {
+
                 $ch = new CurlService($url, $this->timeout, $token, $this->logger);
                 $result = $ch->process();
-                $this->cache->setCacheItem($cacheKey, $result);
+                if ($result['httpCode'] === 200) {
```

Comment:
I think we should update this line to 
`if ($result['httpCode'] === 200 && $result['response'] !== false && empty($result['errorMsg'])) {`
to avoid this case (a 200 response but with a timeout error) : 
![image](https://github.com/hove-io/NavitiaComponent/assets/3872388/86a5ff90-1992-48af-bfde-e03510e6d1ce)